### PR TITLE
moved changes from flow-devel

### PR DESCRIPTION
**Pull request by AboudyKreidieh**

### CODE REVIEW

File: scripts/ray_autoscale.yaml [hunk 1]
```diff
@@ -68,17 +68,17 @@ worker_nodes:
...
 }
 
 setup_commands:
     # checkout your desired branch on all worker nodes
     # - cd learning-traffic && GIT_SSH_COMMAND="ssh -i ~/ray_bootstrap_key.pem" git fetch
-    - cd /tmp && cp foo2 ~/ray_autoscaler_key.pem
-    - cd flow-devel && GIT_SSH_COMMAND="ssh -i ~/ray_autoscaler_key.pem" git fetch  && GIT_SSH_COMMAND="ssh -i ~/ray_autoscaler_key.pem" git checkout `cat /tmp/foo`
+    - cd /tmp && cp ray_autoscaler_key ~/ray_autoscaler_key.pem
+    - cd flow-devel && GIT_SSH_COMMAND="ssh -i ~/ray_autoscaler_key.pem" git fetch  && GIT_SSH_COMMAND="ssh -i ~/ray_autoscaler_key.pem" git checkout `cat /tmp/path`
     - cd ray && git fetch eugene_upstream && git checkout add_ars && git pull
```
Comment (cathywu) [inline scripts/ray_autoscale.yaml]: Same here @eugenevinitsky

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ See [our website](https://berkeleyflow.github.io) for more information on the ap
 
 - [Documentation](https://berkeleyflow.readthedocs.org/en/latest/)
 - [Installation instructions](http://berkeleyflow.readthedocs.io/en/latest/flow_setup.html)
-- [Tutorial](http://berkeleyflow.readthedocs.io/en/latest/tutorial.html)
+- [Tutorials](https://github.com/berkeleyflow/flow/tree/master/tutorials)
```
Comment (eugenevinitsky) [inline README.md]: What's happening here?

Comment (AboudyKreidieh) [reply README.md]: we're linking to the tutorials page on github instead of the old tutorial which is outdated and has been deleted

Comment (eugenevinitsky) [reply README.md]: kk

File: scripts/ray_autoscale.yaml [hunk 3]
```diff
@@ -5,11 +5,11 @@ cluster_name: test #<YOUR NAME>
 
 # The minimum number of workers nodes to launch in addition to the head
 # node. This number should be >= 0.
-min_workers: 2 #<NUM WORKERS IN CLUSTER>
+min_workers: 0 #<NUM WORKERS IN CLUSTER>
 
 # The maximum number of workers nodes to launch in addition to the head
 # node. This takes precedence over min_workers.
-max_workers: 2
+max_workers: 0
```
[the remark this answers is not in the document]
Comment (eugenevinitsky) [reply scripts/ray_autoscale.yaml]: This prevents it from auto-scaling which I think would be undesired behavior.

File: scripts/ray_autoscale.yaml [hunk 4]
```diff
@@ -68,17 +68,17 @@ worker_nodes:
 
 file_mounts: {
 # path to your repo and the desired branch name
-#"/tmp/foo": "<PATH TO LEARNING TRAFFIC>/.git/refs/heads/<BRANCH NAME>",
-#"/tmp/foo2": "<PATH TO RAY_AUTOSCALER_KEY>",
-"/tmp/foo": "/Users/eugenevinitsky/Desktop/Research/Bayen/Code/rllab-multiagent/learning-traffic/.git/refs/heads/ars_runner",
-"/tmp/foo2": "/Users/eugenevinitsky/.ssh/ray-autoscaler_1_us-west-1.pem",
+#"/tmp/path": "<PATH TO LEARNING TRAFFIC>/.git/refs/heads/<BRANCH NAME>",
+#"/tmp/ray_autoscaler_key": "<PATH TO RAY_AUTOSCALER_KEY>",
+"/tmp/path": "/Users/eugenevinitsky/Desktop/Research/Bayen/Code/rllab-multiagent/learning-traffic/.git/refs/heads/ars_runner",
```
[the remark this answers is not in the document]
Comment (eugenevinitsky) [reply scripts/ray_autoscale.yaml]: A commit is coming. Noted.

**Pull request merged**